resources/examples/manager/SampleAssetManager/tests/fixtures.py

```diff
@@ -39,11 +39,5 @@
                 kField_EntityReferencesMatchPrefix: "sam:///"
             }
         }
-    },
-    "Test_managementPolicy": {
-        "test_calling_with_read_context_and_entity_reference": { "valid_entity_reference": dummy_ref },
```

Comment:
[consider][minor] `dummy_ref` is now unused and can be removed.

Reply:
Bah, sorry. Removed [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/233/commits/3c5b12291d307fa5942fe7f81a595a670af7a3de).